daemon/logger/splunk/splunk.go

```diff
@@ -0,0 +1,256 @@
+// +build linux
```

Comment:
@glennblock - Would be happy to verify, just not immediately, busy getting the last few pieces in for technical preview 4 of Windows Server 2016. Drop me a line.